assets/js/admin.js

```diff
@@ -101,6 +102,12 @@ const Admin = {
           allowClearEnabled = false;
         }
 
+        if (select.attr('data-sonata-select2-allow-tags') === 'false') {
+          allowTags = false;
+        } else {
+          allowTags = true;
+        }
```

Comment:
By default `data-sonata-select2-allow-tags` won't be set so it won't be false, so you set allowTags as true.

It should stay false by defaut no ?

Reply:
Good catch! We updated the pr to keep the default behaviour.Good catch! We updated the pr to keep the default behaviour.